go.sum

```diff
@@ -1,6 +1,9 @@
+9fans.net/go v0.0.0-20181112161441-237454027057 h1:OcHlKWkAMJEF1ndWLGxp5dnJQkYM/YImUOvsBoz6h5E=
```

Comment:
slightly unrelated note : how do we want to handle the go.sum ?

Reply:
I guess we'll ave to see how it's done by other people...